app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable, which means it will be shared across all instances of the `Person` class. This might lead to unexpected behavior if you intend to have separate lists of people for different contexts. Consider whether this is the intended design.